cli/hzn.go

```diff
@@ -407,7 +407,7 @@ Environment Variables:
 	deploycheckLong := deploycheckCmd.Flag("long", msgPrinter.Sprintf("Show policies and userinput used for the compatibility checking.")).Short('l').Bool()
 	policyCompCmd := deploycheckCmd.Command("policy", msgPrinter.Sprintf("Check policy compatibility."))
 	policyCompNodeArch := policyCompCmd.Flag("arch", msgPrinter.Sprintf("The architecture of the node. It is required when -n is not specified. If omitted, the service of all the architectures referenced in the deployment policy will be checked for compatibility.")).Short('a').String()
-	policyCompNodeType := policyCompCmd.Flag("node-type", msgPrinter.Sprintf("The node type. The valid values are 'device' and 'cluster'. The default is 'device'.")).Short('t').Default("device").String()
+	policyCompNodeType := policyCompCmd.Flag("node-type", msgPrinter.Sprintf("The node type. The valid values are 'device' and 'cluster'. The default value is the type of node provided by -n (current registered device if nothing is provided).")).Short('t').String()
```

Comment:
Please change the last part of the text to:
"The default value is the type of the node provided by -n  or current registered device, if omitted."
Same for the other 2 places.

Reply:
Changed the description, thanks